tests/unit_tests/flow_variable_tests/auxiliary_variable_test.cpp

```diff
@@ -104,15 +104,13 @@ pcout<<" Grid Index"<<igrid<<std::endl;
 #endif
 //straight
     dealii::GridGenerator::hyper_cube(*grid, left, right, true);
-#if PHILIP_DIM!=1
 	std::vector<dealii::GridTools::PeriodicFacePair<typename dealii::parallel::distributed::Triangulation<PHILIP_DIM>::cell_iterator> > matched_pairs;
```

Comment:
The `#if PHILIP_DIM!=1` should be here since `dealii::parallel::distributed::Triangulation<PHILIP_DIM>` cannot be used for `PHILIP_DIM==1`.

Reply:
The test gets run with mpirun -n 1 explicitly. Ok I'll make the change.